Foundation/NSNotificationQueue.swift

```diff
@@ -31,6 +33,16 @@ public class NSNotificationQueue : NSObject {
     internal let notificationCenter: NSNotificationCenter
     internal var asapList = NSNotificationList()
     internal var idleList = NSNotificationList()
+    internal lazy var idleRunloopObserver: CFRunLoopObserverRef = {
+        return CFRunLoopObserverCreateWithHandler(kCFAllocatorDefault, CUnsignedLong(1) << 5, true, 0) {[weak self] observer, activity in
```

Comment:
the `1 << 5` is not preferred; use kCFRunLoopBeforeWaiting instead and for mac export the constants similarly to the other enum work-arounds